vlab-research/fly
# Feature/abstract business logic from UI component

### What changes have you made?
- moved the business logic from the `Form.svelte` component to `responseStore.js` by creating a `ResponseStore` class
- set up unit tests for the following class methods: `snapshot` and `getQa`
- `snapshot` should return an array made up of the current `ref` (same as the field) and the current `fieldValue `
- `getQa` should return an array of `snapshot` arrays

### Which issue(s) does this PR address?
#63

### How to test
- `npm test` to check that all tests are passing
- `npm run dev` on `localhost:5000` and use the browser to submit test answers to the survey. 
- You can use `console.log` on the `getQa` method to check that all questions and answers are being "stored" in an array
